tutorcairn/templates/cairn/apps/clickhouse/migrations.d/0009_add_graded_column_to_course_blocks.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE course_blocks
+ADD COLUMN graded String;
```

Comment:
Why use a `String` here when we could use a boolean?

Reply:
`====================== course-v1:arbisoft+SE002+2022_S2 test2
Traceback (most recent call last):
  File "/openedx/scripts/importcoursedata.py", line 99, in <module>
    main()
  File "/openedx/scripts/importcoursedata.py", line 35, in main
    import_course(course.id)
  File "/openedx/scripts/importcoursedata.py", line 43, in import_course
    values = [
  File "/openedx/scripts/importcoursedata.py", line 44, in <listcomp>
    sql_format(
  File "/openedx/scripts/importcoursedata.py", line 82, in sql_format
    args = [sql_escape_string(arg).decode() for arg in args]
  File "/openedx/scripts/importcoursedata.py", line 82, in <listcomp>
    args = [sql_escape_string(arg).decode() for arg in args]
TypeError: a bytes-like object is required, not 'bool'`

Reply:
I was getting an error while inserting bool value in the table in importcoursedata.py file.
below query throws an error 
`insert_query = sql_format(
            "INSERT INTO course_blocks (course_id, block_key, block_id, position, display_name, full_name, graded) VALUES "
        )`

Reply:
It will work if you write:

```
"true" if child.graded else "false"
```

Also, please add a default value:

```
ALTER TABLE course_blocks ADD COLUMN graded Boolean DEFAULT false;
```